topics/variant-analysis/tutorials/variant-calling/tutorial.md

```diff
@@ -0,0 +1,605 @@
+---
+layout: tutorial_hands_on

```

Comment:
Since this is doing variant callling from the command line, I think it is better off in the data science topic.

Reply:
Thank you! As you said, the tutorial moved under the "data-science" topic.